recipes/itango/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "itango" %}
+{% set version = "0.1.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 30346418242383484cde347abc93d81fde0147f2f49de71d3edf05a5eb73d098
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  entry_points:
+  # As the package is noarch, we can't use py2k / py3k selector
+  # Create both itango and itango3 entry points
```

As conda-forge doesn't support python 2 anymore py2k won't be build regardless so...
```suggestion
  noarch: python
  entry_points:
```

---

I initially set the recipe as `noarch: python` but it's not possible because `pytango` (run dependency) is only available on Linux so far. So this package should also be Linux only.

---

It's fine for a noarch package to be missing dependencies on Windows or macOS. The solver will then fail saying that `pytango` is not available and if anyone ever adds more platforms to `pytango` then `itango` will just start working. 

The staged recipes CI will fail for Windows and macOS but that won't prevent this PR from being merged.

---

Good to know. Thanks for the explanation!